src/crud/__init__.py

```
@@ -0,0 +1,87 @@
+    "clone_session",
+    "remove_peers_from_session",
+    "get_peers_from_session",
+    "get_session_peer_configuration",
+    "set_peers_for_session",
+    "get_peer_config",
+    "set_peer_config",
+    # Workspace
+    "get_or_create_workspace",
+    "get_all_workspaces",
+    "update_workspace",
+]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)